tests/common/test_statistics_aggregator.py

```diff
@@ -104,70 +112,149 @@ class MinMaxTestParameters:
         ref_max_val: Union[np.ndarray, float]
         ref_min_val: Union[np.ndarray, float]
 
+    MEDIAN_MINMAX =\
```

Comment:
What do you think about extending RangeEstimatorParametersSet by MEAN_QUANTILE, MEAN_NO_OUTLIERS_MINMAX  and MEDIAN_MINMAX?

Reply:
Done